.env.example

```diff
@@ -7,3 +7,7 @@ LOG_LEVEL=debug
 
 # Go to https://smee.io/new set this to the URL that you are redirected to.
 WEBHOOK_PROXY_URL=
+
+#NODE_ENV=production
+#GHE_HOST=Github-Enterprise-Domain
+#PRIVATE_KEY_BASE64="$(cat ./*.private-key.pem | base64)"
```

Comment:
[Question] Is it needed?

Reply:
yes , you should provide your GHE name and also the private generated from the github app for authentication

Reply:
https://github.com/probot/probot/blob/master/docs/deployment.md

Reply:
@elsayed-atef I got it. Thanks